src/Menu.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+
+import { Box, IBoxProps } from './Box';
+import { Flex } from './Flex';
+import { Icon, IIcon } from './Icon';
+
+// TODO allow dividers in the menu
+// TODO allow custom renderMenu
+// TODO allow custom renderMenuItem
+// TODO allow custom renderTrigger to make this a popup type component
```

Comment:
Good start! Updated Menu to take an items props rather than freeform children so that we can enforce consistency. Eventually we want to update to support custom render functions, I added some todo comments as reminders. Another thing we probably want to support is a renderTrigger so that menus could work as popups/dropdowns, but this should be in another story!